tap_shopify/schemas/definitions.json

```diff
@@ -806,6 +836,191 @@
           "null",
           "integer"
         ]
+      },
+      "presentment_title": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "presentment_variant_title": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "user_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "variant_price": {
+        "type": [
+          "null",
+          "string"
+        ],
+        "format": "singer.decimal"
+      },
+      "duties": {},
```

Comment:
In the Shopify API [doc](https://shopify.dev/api/admin-rest/2022-01/resources/refund#resource-object) the `duties` is coming as an object, but from the API response, it is replicated as a list. Hence, used an empty schema here, to accept any value.